src/lib/components/Strip/Strip.scss

```diff
@@ -0,0 +1,72 @@
+@import 'vanilla-framework/scss/base';
+@import 'vanilla-framework/scss/patterns_strip';
+@import 'vanilla-framework/scss/patterns_grid';
+@import 'vanilla-framework/scss/grid/shelves-grid';
+@include vf-b-typography;
+@include vf-p-strip;
+@include vf-p-grid;
+@include vf-p-grid-modifications;
+
+@mixin default-vertical-spacing {
+  margin-top: 1.25rem;
+
+  @media screen and (min-width: #{$breakpoint-medium}) {
+    margin-top: 1.75rem;
+  }
+
+  @media screen and (min-width: #{$breakpoint-large}) {
+    margin-top: $sp-x-large;
+  }
+}
+
```

Comment:
Let's just include the vertical spacing from Vanilla here. It's likely to change soon so will get it of date quickly.